topics/transcriptomics/tutorials/ref-based/tutorial.md

```diff
@@ -1443,15 +1447,17 @@ You should obtain something similar to:
 
 > ### {% icon question %} Questions
 >
-> 1. Do you observe anything in the clustering of the samples and the genes?
-> 2. What changes if you regenerate the heatmap and select `Plot the data as it is` in *"Advanced - log transformation"*/*"Data transformation"*?
-> 3. How could you generate a heatmap of normalized counts for all up-regulated genes with absolute fold change > 2?
+> 1. What does the X-axis of the heatmap represent? What about the Y axis?
+> 2. Do you observe anything in the clustering of the samples and the genes?
+> 3. What changes if you regenerate the heatmap, this time selecting `Plot the data as it is` in *"Advanced - log transformation"*/*"Data transformation"*?
+> 4. How could you generate a heatmap of normalized counts for all up-regulated genes with absolute fold change > 2?
 >
 > > ### {% icon solution %} Solution
 > >
-> > 1. The samples are clustering by treatment.
-> > 2. The scale changes and the differences between the genes are not visible anymore.
-> > 3. Extract the genes with $$log_{2} FC$$ > 1 (filter for genes with `c3>1` on the summary of the differentially expressed genes) and run **heatmap2** {% icon tool %} on the generated table.
+> > 1. The X-axis shows the 7 samples, together with a dendrogram representing the similarity between their patterns of gene expression. The Y-axis shows the 130 differentially expressed genes, likewise with a dendrogram representing the similarity between the patterns of gene expression. <!-- hope this is correct -->
```

Comment:
I found it a bit unclear what the heatmap represents, particularly the unlabelled Y-axis, so added an extra sentence here - I hope it's more or less accurate.